doc/index.rst

```diff
@@ -78,6 +78,22 @@ PennyLane Documentation
         </div>
     </div>
 
+Community
+---------
+We love having a diverse and engaged community that uses PennyLane and contributes to it. 
+We invite you to explore the different resources and events which are designed to help you learn 
+quantum computing and have fun in the process.
+
```

Comment:
```suggestion
```
Adding a new section to the main landing page (in particular, right at the top) is out of scope for the goal here (to just add the ticket)